src/wmtk/TriMeshOperationExecutor.cpp

```diff
@@ -367,6 +367,101 @@ void TriMesh::TriMeshOperationExecutor::replace_incident_face(
 }
 
 Tuple TriMesh::TriMeshOperationExecutor::split_edge()
+{
+    if (!m_mesh.multi_mesh_manager.is_parent_mesh()) {
+        return split_edge_single_mesh();
+    }
+    else
+    {
+        bool is_boundary_on_parent = m_mesh.is_boundary(m_operating_tuple);
+        
+        std::vector<Tuple> vec_t_child;
```

Comment:
vector so plural variable name? and just specify what things are. i have no idea what a vec_t is
`child_tuples`?